extensions/amp-apester-media/0.1/amp-apester-media.js

```diff
@@ -109,6 +108,9 @@ class AmpApesterMedia extends AMP.BaseElement {
 
   /** @override */
   viewportCallback(inViewport) {
+    if (this.intersectionObserverApi_) {
+      this.intersectionObserverApi_.onViewportCallback(inViewport);
```

Comment:
I think you also need to initialize `this.intersectionObserverApi_` in the constructor. =)

Reply:
Good Point. Thanks=)